db/schema.rb

```diff
@@ -76,6 +76,17 @@
 
   add_index "linksets", ["rubygem_id"], name: "index_linksets_on_rubygem_id", using: :btree
 
+  create_table "log_tickets", force: :cascade do |t|
+    t.string   "key"
+    t.string   "directory"
+    t.integer  "backend",    default: 0
```

Comment:
@arthurnn: IMHO, having a facade wrapper around 3rd-party APIs is great, so that's a big :+1: for this PR and `RubygemFs`. It's definitely useful to test with local file; and the pattern I was using of stubbing out AWS API calls is cumbersome.
 
OTOH, adding the `backend` column to db schema to support a testing feature is a smell to me. And the current implementation leads to ​*more*​ test cases, with both the stubbed-out S3 backend, and the local FS backend.

So my pie-in-the-sky ideal would be a way to support S3 stubbing/local files in dev & test envs, without needing the database.